### DISCUSSION TITLE
How can I use the house_consumption_power value in Grafana?

### DISCUSSION THREAD

**Discussion by tgrymatt** [wlcrs/huawei_solar#620] [General]
I am using the "Power Flow Plus Card" described here: https://github.com/wlcrs/huawei_solar/wiki/Visualisations#power-flow-plus-card

But I also want to create a cool and informational Grafana Dashboard with this values. How can I use the "house_consumption_power" value in Grafana? I tried it with the influx query:

SELECT mean("value") AS "mean_value" FROM "homeAssistant"."autogen"."state" WHERE time > :dashboardTime: AND time < :upperDashboardTime: AND "entity_id"='house_consumption_power' GROUP BY time(:interval:) FILL(null)

but this is not working. I always get "Your query or command is syntactically correct but returned no results".

Can you help me?

**Comment by nurzen** [comment]
i think the problem is that the value is not saved in the influxdb. ha calculates it when you refresh the card. 
three possible solutions come to my mind
- you make a measurement in ha where you calculate the value which is then transported to influxdb
- you calculate it in influxdb on the fly in the query
- you let grafana do the calculation by combining different queries
